scalardb-analytics-spark-sample/docker-compose.yml

```diff
@@ -55,50 +55,52 @@ services:
       interval: 1s
       timeout: 1s
       retries: 10
-      start_period: 5s
+      start_period: 10s
 
-  backend-dynamodb:
-    image: amazon/dynamodb-local:1.21.0
+  scalardb-mysql:
+    image: mysql:8.0.36
```

Comment:
[trivial] How about just `8.0` to use the latest patch version?

Reply:
Sure. I'll change the version later.

Reply:
Addressed by https://github.com/scalar-labs/scalardb-samples/pull/75/commits/319f75490c0e3b4d09037a73561762132ac8ab58